statusHandler/presenter/instanceInfoGetters.go

```diff
@@ -1,13 +1,15 @@
 package presenter
 
 import (
-	"math"
 	"math/big"
 	"strings"
 
 	"github.com/ElrondNetwork/elrond-go/core"
 )
 
+// denomination is equals with 10^-4
```

Comment:
denomination is _equal_ _to_ 10^-4

Reply:
This constant was moved in file economics.toml.